findsecbugs-test-kotlin/src/test/kotlin/com/h3xstream/findsecbugs/injection/Slf4jSample.kt

```diff
@@ -0,0 +1,29 @@
+package com.h3xstream.findsecbugs.injection
+
+import org.slf4j.Logger
+
+class Slf4jSample {
+
+    fun slf4j(log: Logger, tainted: DataClass, tainted2: String) {
+        val safe = ""
+        //Unsafe
+        log.info(tainted.input)
+        log.info(tainted.input, safe)
+        log.info(safe, tainted.input)
+        log.info(safe, arrayOf<Any>(tainted.input!!))
```

Comment:
The detectors think this line is safe. From what I understood after much debugging, is that Kotlin compiles that line in several different instructions (because of double `!`), including an if to see if the String is actually not null. Somehow the detector think this is a SAFE instruction when in reality it should be UNKNOWN. 

I didn't manage to find a solution for this one, though. So in the test I skip this line from the assertions. If you know a solution for this problem, let me know.

Reply:
I have created an issue: https://github.com/find-sec-bugs/find-sec-bugs/issues/432